leveldown.js

```diff
@@ -9,7 +9,9 @@ function LevelDOWN (location) {
     return new LevelDOWN(location)
   }
 
-  AbstractLevelDOWN.call(this, location)
+  AbstractLevelDOWN.call(this)
+
+  this.location = location
```

Comment:
We need a type check here (https://github.com/Level/abstract-leveldown/blob/8ca1de858a6c98d9de4eb1063aba670d57df7afa/abstract-leveldown.js#L14-L16)